.github/workflows/unittests.yml

```diff
@@ -89,9 +90,9 @@ jobs:
         with:
           python-version: 3.7
       - name: Install dependencies
-        run: pip install tensorflow==${{matrix.tf-version}} larq~=0.9.1 larq_zoo==1.0.b3 tensorflow_datasets==1.3.2 -e . --no-cache
+        run: pip install tensorflow==${{matrix.tf-version}} larq~=0.9.1 larq_zoo==1.0.b3 tensorflow_datasets==1.3.2 packaging --no-cache
       - name: Run Converter test
-        run: python larq_compute_engine/mlir/python/converter_test.py
+        run: PYTHONPATH=./ python larq_compute_engine/mlir/python/converter_test.py
```

Comment:
Is the `PYTHONPATH` thing set by Github Actions?

Reply:
This is so we don't need to install compute engine via pip. See above.